akka-projection-core/src/main/scala/akka/projection/internal/HandlerRecoveryImpl.scala

```diff
@@ -171,7 +171,7 @@ import akka.projection.StatusObserver
                 retries + 1,
                 exception)
             }
-            retried.recoverWith(_ => futDone)
+            retried.recoverWith { case _ => futDone }
```

Comment:
is that not allowed in 2.13?

Reply:
You man 2.12. 
Yes, this is not allowed.